scripts/python/tests/test_split_git_commit.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from typing import Callable
+
+from git import Repo
+# from pytest_mock.plugin import MockerFixture
```

Comment:
Was this commented out line intentional?

Reply:
Yes, the test script does not currently use [pytest-mock](https://pypi.org/project/pytest-mock/) but will likely do when more tests are added in the future.